command/sub8_launch/config/rise_6dof.yaml

```diff
@@ -1,8 +1,14 @@
-accel_feedforward: 0, 0, 0, 0, 0, 1
-alpha: 0.3, 0.3, 0.3, 0.3, 0.3, 0.3
-beta: 0, 5, 0, 0, 0, 0
-k: 3, 3, 3.0, 1.3, 1.3, 2.0
-ks: 114, 114, 171, 20, 50, 50
-two_d_mode: false
+# RISE terms:
```

Comment:
You know, instead of zeroing these out why not just make use_rise false? Seems like that was the intended use of that flag lol.